Transmission.xcodeproj/project.pbxproj

```diff
@@ -3298,7 +3300,10 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				CLANG_ENABLE_OBJC_ARC = YES;
 				ENABLE_HARDENED_RUNTIME = YES;
-				FRAMEWORK_SEARCH_PATHS = macosx;
+				FRAMEWORK_SEARCH_PATHS = (
+					macosx,
+					"$(PROJECT_DIR)/macosx",
```

Comment:
This change is confusing. Looks to me like it's now 2 elements pointing to the same directory.